configure.ac

```diff
@@ -75,7 +75,8 @@ AC_PROG_OBJC
 AC_PROG_LN_S
 AC_PROG_RANLIB
 AM_PROG_AR
-AX_PROG_CC_FOR_BUILD
+# Should be AX_PROG_CC_FOR_BUILD -- except Steam builds occur in a paleozoic environment where that doesn't work.
```

Comment:
Wouldn't it be possible to provide this macro to the steam environment or is that even more hackey?